src/simcore_service_deployment_agent/auto_deploy_task.py

```diff
@@ -158,6 +158,14 @@ async def generate_stack_file(app_config: Dict, git_task: GitUrlWatcher) -> Path
         raise ConfigurationError(
             "The stack file {} does not exist".format(stack_file.name)
         )
+    if (
+        stack_file.stat().st_size == 0
+    ):  # Filesize check via https://stackoverflow.com/a/55949699
+        raise ConfigurationError(
+            "The stack file {} is empty (0 bytes). Likely your console call to create it failed.".format(
```

Comment:
suggestion what about refactoring both ifs in one go?
```python
if not stack_file.exists() or not stack_file.state().st_size:
    raise ConfigurationError(f"The generation of {stack_file=} failed using {stack_recipe_cfg['command']}")
```

Reply:
done, note the typo on your suggestion: `state()` should be `stat()`

thanks a lot!